src/app/components/login/InviteNewAccount.js

```diff
@@ -110,59 +89,61 @@ const InviteNewAccountComponent = ({ user }) => {
                   alt={appNameHuman}
-                  width="120"
-                  className={['login__icon', classes.logo].join(' ')}
                   src={stringHelper('LOGO_URL')}
+                  width="120"
                 />
               )}
             </FormattedGlobalMessage>
-
-            <Typography component="div" align="center" className={classes.topMargin}>
-              <FormattedMessage
-                id="inviteNewAccount.invitedBy"
-                defaultMessage="{name} has invited you to join the workspace"
-                description="Message to the current user about who has invited them to join this workspace"
-                values={{
-                  name: teamUser?.invited_by?.name,
-                }}
-              />
-            </Typography>
-            <Typography component="div" align="center" className={classes.bold} paragraph>
-              {teamUser?.team?.name}
-            </Typography>
-            <Typography component="div" align="center" paragraph>
+            <div className={styles['login-form-invited']}>
+              <p>
+                <FormattedMessage
+                  id="inviteNewAccount.invitedBy"
+                  defaultMessage="{name} has invited you to join the workspace:"
+                  description="Message to the current user about who has invited them to join this workspace"
+                  values={{
+                    name: teamUser?.invited_by?.name,
+                  }}
+                />
+                <br />
+                <strong>{teamUser?.team?.name}</strong>
+              </p>
               <FormattedHTMLMessage
+                tagName="p"
                 id="inviteNewAccount.createMessage"
-                defaultMessage="You need to create an account for <b>{email}</b>"
+                defaultMessage="You need to create an account for <strong>{email}</strong>"
                 values={{
                   email: user.email,
                 }}
                 description="Inform your to create a new account for signup"
               />
-            </Typography>
-
-            <Message message={message} />
+            </div>
+            {message &&
+              <Alert
+                className={styles['login-form-alert']}
+                content={message}
+                variant="error"
+              />
+            }
 
-            <div className="login__email">
+            <div className={inputStyles['form-fieldset']}>
               <TextField
-                margin="normal"
-                fullWidth
-                variant="outlined"
-                type="email"
-                name="email"
+                required
+                componentProps={{
+                  type: 'email',
+                  name: 'email',
+                }}
                 value={email}
-                className="login__email-input"
+                className={cx('login__email-input', inputStyles['form-fieldset-field'])}
```

Comment:
Are we dropping the `int-` prefix from BEM classes? Just want to make sure we're all on the same page

Reply:
no, will add it back it, I just didn't change the class name, but you are right, good opportunity to "upgrade"